layers/meta-resin-edison/recipes-bsp/u-boot/files/edison.env

```diff
@@ -2,18 +2,18 @@
 # Main part
 
 # Partition definition
-partitions=uuid_disk=${uuid_disk};name=u-boot0,start=1MiB,size=2MiB,uuid=${uuid_uboot0};name=u-boot-env0,size=1MiB,uuid=${uuid_uboot_env0};name=u-boot1,size=2MiB,uuid=${uuid_uboot1};name=u-boot-env1,size=1MiB,uuid=${uuid_uboot_env1};name=factory,size=1MiB,uuid=${uuid_factory};name=panic,size=24MiB,uuid=${uuid_panic};name=resin-boot,size=40MiB,uuid=${uuid_resin_boot};name=resin-root,size=308MiB,uuid=${uuid_resin_root};name=resin-updt,size=308MiB,uuid=${uuid_resin_updt};name=resin-conf,size=20MiB,uuid=${uuid_resin_conf};name=resin-data,size=-,uuid=${uuid_resin_data};
+partitions=uuid_disk=${uuid_disk};name=u-boot0,start=1MiB,size=2MiB,uuid=${uuid_uboot0};name=u-boot-env0,size=1MiB,uuid=${uuid_uboot_env0};name=u-boot1,size=2MiB,uuid=${uuid_uboot1};name=u-boot-env1,size=1MiB,uuid=${uuid_uboot_env1};name=factory,size=1MiB,uuid=${uuid_factory};name=panic,size=24MiB,uuid=${uuid_panic};name=resin-boot,size=40MiB,uuid=${uuid_resin_boot};name=resin-rootA,size=308MiB,uuid=${uuid_resin_rootA};name=resin-rootB,size=308MiB,uuid=${uuid_resin_rootB};name=resin-state,size=20MiB,uuid=${uuid_resin_conf};name=resin-data,size=-,uuid=${uuid_resin_data};
```

Comment:
Right. I was talking about the actual raw image. It would be clearer to be called resin-state.img and not state.img. As well it would be consistent with meta-resin.